### PR TITLE
Trial codes content

### LINKED ISSUE #6358: CC: Coupon codes
AT: https://airtable.com/tblD3oZPLJgGhCmch/viwfjhxZjwRzBXeag/recniHirBsIcTLG1E?blocks=hide

Design doc: https://docs.google.com/document/d/1NENgE8XXHT4A06VVz_dH9FWJ9jWhi1405_QxTALGCgI/edit?usp=sharing_eip&ts=5e28bb37&urp=gmail_link

### PR DESCRIPTION
**Pull request by Amruta-Ranade**
Closes #6358 

cc @davidmargolin for review (GH won't let me assign the PR to you for review)

### CODE REVIEW

File: v19.2/cockroachcloud-console-access-management.md [hunk 1]
```diff
@@ -97,3 +97,11 @@ As a Console Admin, you can set up and manage billing for the Organization.
 1. On the **Billing** page, click the pen icon for the **Billing email address** field.
 2. On the **Edit payment method** page, enter the new email address at which you want get invoices for the Organization.
 3. Click **Add card**.
+
+#### Check validity of trial codes
+
+If you had used a CockroachCloud trial code while [creating a cluster](cockroachcloud-create-your-cluster.html#step-6-enter-your-billing-details), you can check the code expiration details on the **Billing page**.
+
+{{site.data.alerts.callout_info}}
+Make sure that you [delete your trial cluster](cockroachcloud-cluster-management.html#delete-cluster) before the trial expires. Your credit card will be charged after the trial ends. You can check the validity of the code on the [Billing](cockroachcloud-console-access-management.html#manage-billing-for-the-organization) page.
```
Comment (davidmargolin) [inline v19.2/cockroachcloud-console-access-management.md]: Can we say "You can view active trial clusters on the Billing page." ? A customer would be checking if a code is valid during the creation process.

Comment (Amruta-Ranade) [reply v19.2/cockroachcloud-console-access-management.md]: Good point. Reworded the section a bit. LMK if it looks okay now.

Comment (kannanlakshmi) [inline v19.2/cockroachcloud-console-access-management.md]: We don't necessarily want people to delete their trial cluster. I would simply say, if the cluster is not deleted before the trial expires, your card on file will be charged. As written it makes it sound like, users need to delete clusters.

File: v19.2/cockroachcloud-create-your-cluster.md [hunk 2]
```diff
@@ -88,8 +88,12 @@ After entering the cluster name, click **Continue to Payment**.
     {{site.data.alerts.end}}
     You will be billed on the 1st and 15th of every month.
 3. Add your preferred [payment method](cockroachcloud-console-access-management.html#manage-billing-for-the-organization).
-4. Check the **I agree to Cockroach Labs Terms of Service and Privacy Policy** box.
-5. Click **Create cluster**.
+4. (Optional) If you have received a trial code, enter the code in the **Trial code** box.
+      {{site.data.alerts.callout_info}}
+      Make sure that you [delete your trial cluster](cockroachcloud-cluster-management.html#delete-cluster) before the trial expires. Your credit card will be charged after the trial ends. You can check the validity of the code on the [Billing](cockroachcloud-console-access-management.html#manage-billing-for-the-organization) page.
```
Comment (kannanlakshmi) [inline v19.2/cockroachcloud-create-your-cluster.md]: same as above

File: v19.2/cockroachcloud-console-access-management.md (hunk 1 above)
Comment (Amruta-Ranade) [reply v19.2/cockroachcloud-console-access-management.md]: Excellent point. Fixed.

**Pull request merged**